packages/kotti-ui/source/next/kotti-field-select/hooks.ts

```diff
@@ -131,6 +131,44 @@ const useSelectInputSizeFix = <DATA_TYPE extends Values>({
 	})
 }
 
+/**
+ * chrome 66+ shows suggestions for saved autofill data,
+ * through intelligent detection, recognizing "address" fields, for instance,
+ * and suggesting autofills.
+ *
+ * using `autocomplete="off" on `input` of `type="search"` would not work to disable autofill
+ *
+ * HACK to exclude the input field from being auto-fillable
+ * @see {@link https://gist.github.com/niksumeiko/360164708c3b326bd1c8#gistcomment-2032962 }
+ *
+ * could potentially break in even newer versions of chrome.
+ * could be solved by accepting an autocomplete prop
+ * and using tokens for commonly saved autofill fields (e.g. address-line-1)
```

Comment:
Would still argue that this is wrong, as we DO have a `search` field. No matter what the browser autocompletes there, it will be wrong.

The only thing `autocomplete="address-line-1"` does is tell the browser to "please auto complete addresses in this field" which we do not want in a single select search field.